config/locales/api_entreprise/cas_usages_entries.fr.yml

```diff
@@ -317,20 +328,232 @@ fr:
+        use_case_examples:
+          title: null
+          anchor: null
+          content: null
+        list_api:
+          content: |+
+            {:.fr-text--xl}
+            Ce tableau dresse la liste des API utiles pour compléter le [formulaire unique de demande de subventions](https://www.associations.gouv.fr/le-formulaire-unique-de-demande-de-subvention-est-en-ligne.html){:target="_blank"} dont le contenu est détaillé dans le [décrêt n° 2016-1971 du 28 décembre 2016](https://www.legifrance.gouv.fr/loda/id/JORFTEXT000033736810/2022-01-02){:target="_blank"}. <br>Utilisez le tableau pour compléter votre [demande d'habilitation](#request-access). 
+
+            Les API **<span style="color:#1f8d49" class="fr-icon-checkbox-circle-fill" aria-hidden="true"></span> indispensables seront précochées** par défaut dans le formulaire d'accès. 
+            Les API **️<span style="color:#1f8d49" class="fr-icon-checkbox-circle-line" aria-hidden="true"></span> utiles** seront à cocher par vos soins si votre cas d'usage le justifie.
+            Les API **<span style="color:#d64d00" class="fr-icon-close-circle-line" aria-hidden="true"></span> injustifiées** ne peuvent être demandées pour ce cas d'usage.
+
+            {:.fr-highlight.fr-highlight--caution}
+            Lors de votre demande d'habilitation, **vous vous engagez à ne demander que les données strictement nécessaires** à la réalisation de la démarche administrative. Consulter [nos conditions générales d'utilisation](<%= cgu_path %>){:target="_blank"}.
+        users:
+          title: Ils utilisent l’API Entreprise
+          anchor: 'users'
+          content: |+
+            {:.fr-h5}
+            ### Des communes et intercommunalités
+            Alsace collectivité européenne, Corbas, Les Sables-D’olonne, Saint Chamond, Communauté d’agglomération Chauny-Tergnier-La Fère, Communauté urbaine d’Arras, Grand Poitiers communauté urbaine, Vaux-en-Velin, Venissieux.
+
+            <figure class="fr-quote--column ">
+            {:.fr-quote} 
+            >
+            > <blockquote>«&nbsp;La ville attribue des subventions aux associations de son territoire [...], elle a mis en oeuvre un portail de gestion de la relation citoyen dédié au suivi de la relation avec ses associations. Pour s'enregistrer sur ce portail, puis pour pouvoir solliciter une aide de la collectivité, l'association doit justifier de son immatriculation et de l'identité de ses représentants légaux. Le logiciel permet d'interroger l'API Entreprise pour renseigner ces différentes informations par défaut.&nbsp;»
```

Comment:
Suggestion pour que ce soit + simple à lire et "donne envie de lire jusqu'a la fin" . Pour chaque exemple : 
Contexte : Besoin : traiter les demandes de subventions assos
Intégration API entreprise : plateforme GRU/GRC 
Avantages / bénéfices : Gain de temps + données securisées 
J'ai fait çà rapidement mais qu'en penses -tu ?

Reply:
Ici, je suis moins sûre, je pense que c'est utile de reprendre les "mots" des usagers, les textes que j'ai pris, c'est la description qu'ils donnent dans leur demande Datapass.

Je suis vraiment pas pour mettre des bullets points, car c'est déjà ce qu'on donne plus haut.
Si ça te va, je propose de mettre en prod cette première version et d'itérer par la suite sur les témoignages autres éléments.